app/models/account.rb

```diff
@@ -563,7 +563,7 @@ def provider_id_for_audits
   end
 
   def independent_mapping_rules_enabled?
-    provider_can_use?(:api_as_product) || provider_can_use?(:independent_mapping_rules)
+    true
   end
```

Comment:
Then do we even need this method at all?

Reply:
That triggers a LOT of scary changes but yes 😱